bin/auto-pr.ps1

```diff
@@ -21,8 +21,9 @@ $dir = resolve-path $dir
 
 . "$psscriptroot\..\lib\manifest.ps1"
 . "$psscriptroot\..\lib\json.ps1"
+. "$psscriptroot\..\lib\unix.ps1"
 
-if(isUnix) {
+if(is_unix) {
```

Comment:
Maybe change this to `is_mac` and use `elseif(is_unix)`, suggesting favorite package manager to install hub or go to github?